spec/workers/translation_sync_worker_spec.rb

```diff
@@ -29,6 +29,21 @@
           translation.reload.strings["title"]
         }.from(old_title).to(new_title)
       end
+
+      it 'updates strings if workflow has no versions yet' do
+        workflow = create :workflow
+        translation = create :translation, translated: workflow
+        old_string = translation.strings["tasks.shape.question"]
+        workflow.primary_content.strings["shape.question"] = "asdf"
```

Comment:
Aren't workflow translations built from the workflow strings attribute now? Is this needed because the primary contents are clobbering the resource strings attribute in https://github.com/zooniverse/Panoptes/blob/2b1e23182adb5e9b19994d4c24d020cdc6121fb2/app/models/concerns/translation_strings.rb#L27

Reply:
Yep. We should probably stop doing that, but maybe not in this PR.